explorer/client/src/utils/api/DefaultRpcClient.ts

```diff
@@ -22,8 +22,15 @@ import type {
 
 export { Network, getEndpoint };
 
-export const DefaultRpcClient = (network: Network | string) =>
-    new JsonRpcProvider(getEndpoint(network));
+const defaultRpcMap: Map<Network | string, JsonRpcProvider> = new Map();
```

Comment:
This change solved a problem with clients being recreated - that works fine when the client has no state, but a websocket connection adds some state